ocrd_utils/ocrd_utils/logging.py

```diff
@@ -141,6 +141,11 @@ def initLogging():
         logging.getLogger('tensorflow').setLevel(logging.ERROR)
 
     if _overrideLogLevel:
```

Comment:
```suggestion
    if _overrideLogLevel:
        # for existing loggers that won't have getLogger do this
        # (because they were instantiated through logging.getLogger
        #  instead of ours, or come from logging.config.fileConfig),
        # unset log levels so the global override can apply:
```